packages/client/src/components/interface/Navigation.tsx

```diff
@@ -338,7 +337,7 @@ export const NavigationView = (props: IFullProps) => {
     <LeftNavigation
       applicationName={offlineCountryConfiguration.config.APPLICATION_NAME}
       applicationVersion={runningVer}
-      buildVersion={BUILD_VERSION}
+      buildVersion={import.meta.env.VITE_APP_VERSION ?? 'Development'}
```

Comment:
I get this error without ?? 'Development' because LeftNavigation expects a string:
```
Type 'string | undefined' is not assignable to type 'string'
```